bin/create-dotenv

```diff
@@ -61,4 +61,14 @@ EOF
         echo -e "\nLOCALLY_DEPLOYED_SERVICES=harmony-service-example,giovanni-adapter\n" >> .env
     fi
 
+    # set Harmony core services to use the stable image by default for harmony-in-a-box
+    echo "HARMONY_IMAGE=harmonyservices/harmony:stable" >> .env
```

Comment:
Generally we set all the default values to work with harmony in a box (so we would set it to use `stable` for everything and then override if we want to change any of the images to latest).

I think that makes the most sense in this case as well to have the defaults as stable, but we can maybe talk about it at the tech meeting tomorrow.